spec/furoshiki/base_app_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'pathname'
+require 'furoshiki/base_app'
+
+include PackageHelpers
```

Comment:
argh, wouldn't include the PackageHelpers on the top level if possible, you can do so within a describe block, should work (include or extend... one of the 2 works)